tests/test_weighted_mean_estimator.py

```diff
@@ -67,11 +72,13 @@ def basis(L, dtype):
 
 
 @pytest.fixture(scope="module")
-def weights(sim):
+def weights(sim, weights_dtype):
     # Construct simple test weights;
     # one uniform positive and negative weighted volume respectively.
     r = 2  # Number of weighted volumes
-    weights = np.ones((sim.n, r)) / np.sqrt(sim.n)
+    weights = np.ones((sim.n, r), dtype=weights_dtype) / np.sqrt(
```

Comment:
if we're going to change this line, maybe just go to `np.full`.

The sqrt with the typing is pretty unfortunate.

Reply:
The sqrt typing is unfortunate. Either way, changed to using `np.full`. Thanks